api/schemas/views.py

```diff
@@ -23,7 +26,7 @@ class RegistrationSchemaList(JSONAPIBaseView, generics.ListAPIView):
     required_write_scopes = [CoreScopes.NODE_DRAFT_REGISTRATIONS_WRITE]
 
     serializer_class = RegistrationSchemaSerializer
-    view_category = 'registration-schemas'
+    view_category = 'schemas'
```

Comment:
With the `FileMetadataSchema` (or something along those lines) going in soon, this might not be a wise change.